experimental/examples/tree-sample/tree-react-api/src/test/schema.ts

```diff
@@ -0,0 +1,35 @@
+/*!
```

Comment:
This is the unit test copy of the schema, which was previously spread across three files (now concatenated here).

The "unit test" is really just a debugging convenience at this point.